### PR TITLE
Issue 4480: Allow user to specify root folder in package command

### PR DESCRIPTION
**Pull request by hlhoots**
Users need to be allowed to specify the root folder name within the output archive whenever the server package command is executed.

### CODE REVIEW

**Comment by hlhoots** [comment]
#build

**Comment by hlhoots** [comment]
#build

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 1]
```diff
@@ -684,6 +684,10 @@ warning.server.status.invalid.targets=CWWKE0946W: A status request was received
 warning.server.status.invalid.targets.explanation=A user issued a status request with an empty target list on the target option. The request completed but no action was taken.
 warning.server.status.invalid.targets.useraction=Specify on the target option a list of components to check their status. To check the cumulative status of all pause capable components in the server, issue status request without the target option.
 
+warning.package.root.invalid=CWWKE0947W: A --server-root option with empty argument is not allowed when Product Extensions are installed. The default argument of wlp will be used.
```
Comment (rhpeters) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Recommend adding the word "an": "...option with an empty argument".

**Comment by hlhoots** [comment]
Made updates to message per ID review.  Added debug to both TC's.  

#build

**Comment by hlhoots** [comment]
#build

**Comment by hlhoots** [comment]
Last build didnt pickup a change even though it was pushed up.  Re-building.

#build

**Comment by hlhoots** [comment]
#build

**Comment by hlhoots** [comment]
PR for #4480

**Comment by hlhoots** [comment]
#build

**Pull request merged**